Cargo.toml

```diff
@@ -24,6 +24,7 @@ cadence = "0.25"
 docopt = "1.1"
 cloud-storage = "0.6" # 0.7+ includes request 0.11, tokio 1.4
 config = "0.11"
+dashmap = "4.0.2"
```

Comment:
one note about dashmap: it's susceptible to a [theoretical spinlock deadlock situation](https://github.com/xacrimon/dashmap/issues/142).

I'm not very worried about this and there's already a [PR in progress](https://github.com/xacrimon/dashmap/pull/145) to fix it.